src/docs/reference/flutter-cli.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "flutter: The Flutter command-line tool"
+description: "The reference page for using 'flutter' in a terminal window."
+---
+
+The `flutter` command-line tool is how developers (or IDEs on behalf of developers) interact with Flutter.
+
+Here's how you might use the `flutter` tool to create, analyze, test, and run an app:
+
+```terminal
+$ flutter create my_app
+$ cd my_app
+$ flutter analyze
+$ flutter test
+$ flutter run bin/my_app.dart
```

Comment:
This command needs to be different. (Delete the path, leaving just `flutter run`?)